src/openrct2-ui/windows/DebugPaint.cpp

```diff
@@ -27,21 +27,23 @@
 enum WINDOW_DEBUG_PAINT_WIDGET_IDX
 {
     WIDX_BACKGROUND,
+    WIDX_TOGGLE_SHOW_WIDE_PATHS,
     WIDX_TOGGLE_SHOW_BLOCKED_TILES,
     WIDX_TOGGLE_SHOW_SEGMENT_HEIGHTS,
     WIDX_TOGGLE_SHOW_BOUND_BOXES,
     WIDX_TOGGLE_SHOW_DIRTY_VISUALS,
 };
 
 #define WINDOW_WIDTH    (200)
```

Comment:
While you're at it, could you make this box a little wider? It's too narrow to fit most non-English strings.